packages/dashboard-frontend/src/containers/FactoryLoader/index.tsx

```diff
@@ -276,10 +276,21 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
     }
     const { source } = this.factoryResolver.resolver;
     const searchParam = new window.URLSearchParams(this.state.search);
-    const devfileLocationInfo = !source || source === 'repo' ?
-      `${searchParam.get('url')}` :
-      `\`${source}\` in github repo ${location}`;
-    this.setState({ devfileLocationInfo });
+    let resolvedDevfileMessage: string;
+    // source tells where devfile comes from
+    //  - no source: the url to raw content is used
+    //  - repo: means no devfile is found and default is generated
+    //  - any other - devfile is found in repository as filename from the value
+    if (!source) {
+      resolvedDevfileMessage = `Devfile loaded from ${searchParam.get('url')}`;
```

Comment:
> just wondering about the 'is found' vs 'was found' (or even drop the is)

@benoitf thx for feedback. Now we have
```
1. `Devfile loaded from ${searchParam.get('url')}`;
2. `Devfile could not be found in ${searchParam.get('url')}. Applying the default configuration`;
3. `Devfile found in repo ${searchParam.get('url')} as '${source}'. Applying it`;
4. `Devfile could not be found`
```
Any suggestions?

Reply:
no that's fine for me